libnetwork/drivers/bridge/setup_device_linux_test.go

```diff
@@ -92,3 +94,35 @@ func TestGenerateRandomMAC(t *testing.T) {
 		t.Fatalf("Generated twice the same MAC address %v", mac1)
 	}
 }
+
+func TestMTUBiggerThan1500(t *testing.T) {
```

Comment:
For a follow-up we could add a GoDoc to these tests to mention where these specific ranges came from (what's "magical" about Bigger than 1500, or bigger than 64k?

And perhaps combining both to a test-table / sub-tests would allow adding a "docs" to the tests (and more easily allow more cases to be added)